static-instrumenter/maven-plugin/src/test/java/io/opentelemetry/contrib/staticinstrumenter/plugin/maven/AbstractTempDirTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.staticinstrumenter.plugin.maven;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+
+// Workaround for https://github.com/junit-team/junit5/issues/2811
+abstract class AbstractTempDirTest {
+
+  File tempDir;
+
+  @BeforeEach
+  public void before() {
+    try {
+      tempDir = Files.createTempDirectory(this.getClass().getSimpleName()).toFile();
+    } catch (IOException e) {
+      throw new RuntimeException("Problem in the creation of the temp directory", e);
+    }
+  }
```

Comment:
seems ok to add `throws IOException`, doesn't necessarily seem worth friendly message in this case